File: Angular/Dabhi Vipul/Day9/AssignmentDay9/app/student.ts
type Parent = {
    firstname: string
    middlename: string
    lastname: string
    email: string
    qualification: string
    profession: string
    designation: string
    phone: string
  }
  
  type Address = {
    city: string
    state: string
    country: string
    pin: number
  }
  
  type Emergency = {
    relation: string
    phone: number
  }
  
  export interface Student {
    firstname: string
    middlename: string
    lastname: string
    dateofbirth: Date
    placeofbirth: string
    firstlanguage: string
    address: Address
    father: Parent
    mother: Parent
    emergency: Emergency[]
  }

File: Angular/Dabhi Vipul/Day14&15/StudentWebApi/StudentWebApi/Models/Student.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace StudentWebApi.Models
{
    public partial class Student
    {
        public int StudentId { get; set; }
        public string FirstName { get; set; }
        public string MiddleName { get; set; }
        public string LastName { get; set; }
        public DateTime? DateOfBirth { get; set; }
        public string PlaceOfBirth { get; set; }
        public string FirstLanguage { get; set; }
        public int? Address { get; set; }
        public int? Father { get; set; }
        public int? Mother { get; set; }
        public int? Emergency { get; set; }
        public int? Reference { get; set; }

        public virtual Address AddressNavigation { get; set; }
        public virtual Father FatherNavigation { get; set; }
        public virtual Mother MotherNavigation { get; set; }
        public virtual Reference ReferenceNavigation { get; set; }
        public virtual Emergency EmergencyNavigation { get; set; }
    }
}
